specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -1,43 +1,42 @@
 == Transaction Management
 
-This chapter describes the required Java™
-Platform, Enterprise Edition (Java EE) transaction management and
-runtime environment.
+This chapter describes the required Jakarta Enterprise Edition (Jakarta EE)
+transaction management and runtime environment.
 
 Product Providers must transparently support
 transactions that involve multiple components and transactional
-resources within a single Java EE product, as described in this chapter.
-This requirement must be met regardless of whether the Java EE product
+resources within a single Jakarta EE product, as described in this chapter.
+This requirement must be met regardless of whether the Jakarta EE product
 is implemented as a single process, multiple processes on the same
 network node, or multiple processes on multiple network nodes.
 
 If the following components are included in a
-Java EE product, they are considered transactional resources and must
+Jakarta EE product, they are considered transactional resources and must
 behave as specified here:
 
 * JDBC connections
-* JMS sessions
+* Jakarta Messaging sessions
 * Resource adapter connections for resource
 adapters specifying the _XATransaction_ transaction level
 
 === Overview
 
-A Java EE Product that includes both a servlet
-container and an EJB container must support a transactional application
-comprised of combinations of web application components accessing
-multiple enterprise beans within a single transaction. If the Java EE
+A Jakarta EE Product that includes both a servlet
+container and an enterprise bean container must support a transactional
+application comprised of combinations of web application components accessing
+multiple enterprise beans within a single transaction. If the Jakarta EE
 product also includes support for the Connectors specification, each
 component may also acquire one or more connections to access one or more
 transactional resource managers.
 
 For example, in
-_link:#a475[See Servlets/JSP Pages
+_link:#a475[See Servlets/Server Pages
 Accessing Enterprise Beans]_ , the call tree starts from a servlet or
-JSP page accessing multiple enterprise beans, which in turn may access
+server pages accessing multiple enterprise beans, which in turn may access
 other enterprise beans. The components access resource managers via
 connections.
 
-=== [[a475]]Servlets/JSP Pages Accessing Enterprise Beans
+=== [[a475]]Servlets/Server Pages Accessing Enterprise Beans
 
 image:Platform_Spec-8.png[image]
```

Comment:
Is not working for me: https://github.com/eclipse-ee4j/jakartaee-platform/blob/710bc5e80e163b33f3acf47f7a089830d16215d1/specification/src/main/asciidoc/platform/Platform_Spec-8.png - the 404 I was referring to was the image in there, not the entire document

Reply:
Yes, I also get 404. The link in master (https://github.com/eclipse-ee4j/jakartaee-platform/blob/master/specification/src/main/asciidoc/platform/Platform_Spec-8.png) throws 404 too. What should we do?

Reply:
My guess is it is meant to be linked to https://github.com/eclipse-ee4j/jakartaee-platform/blob/master/specification/src/main/asciidoc/images/Platform_Spec-8.png - perhaps it does that during the build or maybe the URL needs images/ in the path?

Reply:
image:images/Platform_Spec-8.png[image] fixed the image issue